vector/src/main/java/im/vector/riotx/features/home/room/detail/readreceipts/DisplayReadReceiptsController.kt

```diff
@@ -40,7 +42,12 @@ class DisplayReadReceiptsController @Inject constructor(private val dateFormatte
                     .matrixItem(it.toMatrixItem())
                     .avatarRenderer(avatarRender)
                     .timestamp(timestamp)
+                    .userClicked { listener?.didSelectUser(it.userId) }
                     .addIf(session.myUserId != it.userId, this)
         }
     }
+
+    interface DisplayReadReceiptsControllerListener {
```

Comment:
Just call it `Listener` because it's an inner interface so it will be prefixed by the class name.